### PR TITLE
fix(import and export): add base64 for password when import and expor…

### PR DESCRIPTION
**Pull request by Rory-Z**
…t data

### CODE REVIEW

**Review by HJianBo — APPROVED** [review]

File: include/emqx_mgmt.hrl [hunk 1]
```diff
@@ -32,4 +32,4 @@
 -define(ERROR14, 114). %% OldPassword error
 -define(ERROR15, 115). %% bad topic
 
--define(VERSIONS, ["1", "3.2", "3.4", "4.0", "4.1", "4.2"]).
\ No newline at end of file
+-define(VERSIONS, ["1", "3.2", "3.4", "4.0", "4.0.13", "4.1", "4.2", "4.2.10"]).
```
Comment (zmstone) [inline include/emqx_mgmt.hrl:35]:
```suggestion
-define(VERSIONS, ["4.0.13", "4.1", "4.2.10"]).
```

Comment (Rory-Z) [reply include/emqx_mgmt.hrl]: Updated

File: src/emqx_mgmt.erl [hunk 2]
```diff
@@ -823,6 +824,30 @@ to_version(Version) when is_binary(Version) ->
...
 
+is_version_supported(Data, Version) ->
+    case { maps:get(<<"auth_clientid">>, Data, [])
+         , maps:get(<<"auth_username">>, Data, [])
+         , maps:get(<<"auth_mnesia">>, Data, [])} of
+        {[], [], []} -> lists:member(Version, ?VERSIONS);
+        _ -> is_version_supported2(Version)
+    end.
+
+is_version_supported2("4.1") ->
```
Comment (k32) [inline src/emqx_mgmt.erl:835]: This clause can be moved inside `try lists:map(...) of` to reduce the number of nested conditions

File: src/emqx_mgmt.erl [hunk 3]
```diff
@@ -823,6 +824,30 @@ to_version(Version) when is_binary(Version) ->
...
+    true;
+is_version_supported2(Version) ->
+    case re:run(Version, "^4.[02].\\d+$", [{capture, none}]) of
```
Comment (k32) [inline src/emqx_mgmt.erl:838]: Perhaps the regexp could be simplified to just `"^(\\d+\\.){2}\\d+$"`, so it only checks format of the version, but not the version itself.

**Review by tigercl — APPROVED** [review]

**Review by terry-xiaoyu — APPROVED** [review]

**Pull request merged**